qiita_pet/handlers/rest/rest_handler.py

```diff
@@ -0,0 +1,28 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from qiita_db.study import Study
+from qiita_db.exceptions import QiitaDBUnknownIDError
+from qiita_pet.handlers.util import to_int
+from qiita_pet.handlers.base_handlers import BaseHandler
+
+
+class RESTHandler(BaseHandler):
+    def fail(self, msg, status):
```

Comment:
Sounds like a great idea! The reason why we had been putting the error message in the body is because we were modeling the API's *etiquette* after GitHub's API, where they always return either the data or a dictionary with **just** a message, and they even return a link to their API's documentation. So what about modifying `QiitaHTTPError` to write the message to the body?

Just for a reference, this is how it looks when you use curl to pull some data and the request fails:

```
(qiita)yoshikivazquezbaeza:qiita@death-march$ curl -ik 'https://127.0.0.1:21174/api/v1/person?name=LabDude&affiliation=knight%20lab2'
HTTP/1.1 404 Not Found
Date: Thu, 30 Mar 2017 17:59:45 GMT
Content-Length: 31
Content-Type: application/json; charset=UTF-8
Server: TornadoServer/3.1.1

{"message": "Person not found"}
```


Thinking more, should we be logging these errors into the database, or is the Tornado logging sufficient?